docs/self-managed/platform-deployment/helm-kubernetes/guides/multi-namespace-deployment.md

```diff
@@ -0,0 +1,125 @@
+---
+id: multi-namespace-deployment
+title: "Multi-namespace deployment"
+description: "Deploy Camunda 8 Self-Managed across namespaces"
+---
+
+You can deploy Camunda 8 Self-Managed across namespaces, where we have a single management deployment (Identity and Web Modeler), and multiple automation deployments (Zeebe, Operate, Tasklist, Optimize).
+
+In this setup each namespace has its own Helm deployment independently, but each one uses a different values file based on if the deployment mode is `management` or `automation`.
+
+The following are three values files, one for the `managment` deployment and two for the `automation` deployment:
+
+## Management deployment
+
+This deployment works with Identity and Web Modeler only and allows another two deployments to authenticate against Keycloak:
```

Comment:
"and allows another two deployments to authenticate against Keycloak" -- I adjusted the wording here. Is this what you meant?

Reply:
@christinaausley  The meaning is a bit different, so I've reworded it.
Could you please give it another look?